### ISSUE TITLE
We should consider disallowing external images in the editor

### ISSUE THREAD

**Issue by burtonator** [hicommonwealth/commonwealth#9216]
## Describe the bug

The image dialog allows us to provide 3rd party image URLs 

![CleanShot 2024-09-12 at 15 15 50@2x](https://github.com/user-attachments/assets/f352bdf0-bb0b-4eb2-a993-51cc6829b893)

There are two main problems here:

1. They can provide **any** URL but it might break. It could be a HTML URL for example.
2. The image might be a valid URL but blocked for us via Content-Security-Policy.
3. The URL could break in the future and we would have no idea why.

I think we should disallow this, and only allow image pasting or manually uploading images via files so they're stored in our S3.  

What do you guys think?

@timolegros @jnaviask @mzparacha @masvelio 

### Initial conditions

**Environment:** 

**Branch/Release version:** 

**Browser:** 

**Wallet:**

**Comment by timolegros** [comment]
I agree. External image URLs should be treated as normal URLs (no rendering). Only images in our S3 bucket should be rendered.

**Comment by masvelio** [comment]
I second to @timolegros comment

**Comment by mzparacha** [comment]
I also agree with @timolegros.

From OP: 

> 3. The URL could break in the future and we would have no idea why.

I think it's fine.

**Comment by burtonator** [comment]
Nice!  Love that we all agree on this :)
